### PR TITLE
Feat/bloblong

### PR DESCRIPTION
**Pull request by Thimey**
Adds bloblong spawn.

Currently bloblong can only be be selected and move. Connection creation will come next

### CODE REVIEW

File: src/Game/Blobs/bloblong/bloblong.ts [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+function switchDirection(dir: 1 | -1) {
+  return dir === 1 ? -1 : 1;
+}
+
+const rotateFin = assign<Context, UpdateEvent>(
+  ({ finRotation, finRotationDir, movement }) => {
+    if (!movement) return {};
+
+    const shouldRotate = movement.pathIndex % 5 === 0;
```
Comment (Oolay) [inline src/Game/Blobs/bloblong/bloblong.ts:73]: 5 is a bit of a magic number here, where does it come from?

File: src/Game/Blobs/bloblong/draw.ts [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+const BLOBLONG_HEAD1_EYE_OFFSET = -30;
+const BLOBLONG_HEAD2_EYE_OFFSET = 30;
+const BLOBLONG_EYE_RADIUS = 1.5;
+
+function makePoint({ x, y }: Point, offset: number, rotation: number) {
+  return {
+    x: x + offset * Math.cos(rotation),
+    y: y + offset * Math.sin(rotation),
+  };
+}
```
Comment (Oolay) [inline src/Game/Blobs/bloblong/draw.ts:42]: Could move this to Math lib. Maybe we could destructure the Point arg inside the function body so we can name the argument for more context -> makePoint(referencePoint: Point,...)

**Review by Oolay — APPROVED** [review]
Great work! Love the fin animation! 🤌

File: src/Game/Blobs/bloblong/bloblong.ts (hunk 1 above)
Comment (Thimey) [reply src/Game/Blobs/bloblong/bloblong.ts]: I moved to a util `multipleOf` so the number has context

**Pull request merged**